files/images/teamspeak_viewer/badges/gamescom_2018_details.svg

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="Layer_1"
+   x="0px"
+   y="0px"
+   viewBox="0 0 64.000001 64.000001"
+   xml:space="preserve"
+   inkscape:version="0.91 r13725"
+   sodipodi:docname="gamescom_2018_details.svg"
+   width="64"
+   height="64"><metadata
+     id="metadata19"><rdf:RDF><cc:Work
+         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title></dc:title></cc:Work></rdf:RDF></metadata><defs
+     id="defs17" /><sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
+     inkscape:window-width="2560"
+     inkscape:window-height="1416"
+     id="namedview15"
+     showgrid="false"
+     inkscape:zoom="9.8333333"
+     inkscape:cx="12"
+     inkscape:cy="12.000002"
+     inkscape:window-x="1920"
+     inkscape:window-y="24"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="Layer_1"
+     fit-margin-top="0"
+     fit-margin-left="0"
+     fit-margin-right="0"
+     fit-margin-bottom="0" /><style
+     type="text/css"
+     id="style3">
+	.st0{fill:#512C83;}
+	.st1{fill:#57ACE0;}
+</style><g
+     id="g5"
+     transform="scale(2.6666667,2.6666667)"><g
+       id="g7"><circle
+         class="st0"
+         cx="12"
+         cy="12"
+         id="ellipse9"
+         r="12"
+         style="fill:#512c83" /></g><g
+       id="g11"><path
+         class="st1"
+         d="m 6,5.9 c 3.3,-3.3 8.7,-3.3 12,0 3.3,3.3 3.3,8.7 0,12 -1.6,1.6 -3.8,2.6 -5.9,2.6 l 0,-2.4 c 1.6,0 3,-0.7 4.2,-1.9 2.4,-2.4 2.4,-6.1 0,-8.5 -2.4,-2.4 -6.2,-2.4 -8.5,0 -2.3,2.4 -2.4,6.1 0,8.5 l 4.3,4.3 0,3.5 L 6,18 6,18 C 2.7,14.7 2.7,9.3 6,5.9 m 3.5,3.6 c -1.4,1.4 -1.4,3.6 0,5 1.4,1.4 3.6,1.4 5,0 1.4,-1.4 1.4,-3.6 0,-5 -1.4,-1.4 -3.7,-1.4 -5,0"
+         id="path13"
+         inkscape:connector-curvature="0"
+         style="fill:#57ace0" /></g></g></svg>
```

Comment:
I miss line breaks

Reply:
Rechecked with original source. The file is untouched from original.